frontend/src/components/menu.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect } from 'react';
+import {
+  View, Text, Dimensions,
+} from 'react-native';
+import TouchableImage from './TouchableImage';
+import buttomImage1 from '../../assets/menu/Buildings.png';
+import buttomImage2 from '../../assets/menu/Troops.png';
+import buttomImage3 from '../../assets/menu/Battle.png';
+import buttomImage4 from '../../assets/menu/Leaderboard.png';
+
+export default function Menu() {
+  const [isPressed, setPressed] = useState('Buildings');
+
+  useEffect(() => { }, [isPressed]);
+
+  function handlePress(name) {
+    setPressed(name);
+    // switch (isPressed) {
+    //   case 'Buildings':
+    //     // TODO
+    //     break;
+    //   case 'Troops':
+    //     // TODO
+    //     break;
+    //   case 'Battle':
+    //     // TODO
+    //     break;
+    //   case 'Leaderboard':
+    //     // TODO
+    //     break;
+    //   default:
+    //     setPressed('Buildings');
+    //     // TODO
+    // }
+  }
+
+  return (
+    <View style={{
+      flexDirection: 'row',
+      width: Dimensions.get('window').width * 0.9,
```

Comment:
what are these magic numbers like 0.9 and 0.22? 
save them in a const with a descriptive variable name which will let the reader know why is it 0.9

Reply:
Since I want my menu component changes depends on the device's size. I just make a 5% margin for it. Same as the height, every button shared 25% of the screen and 3% is for margin.
Do I have to add a comment on my page?